Bottom: 098ad2fe5477176791ee64b82b568dbfe7803c68
Top:    e5a0f6b2064294f7bb3a3605ee33ca977c406064
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 18:44:25 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index d7b5e34..ae0b9fa 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
